R/lasnormalize.r

```diff
@@ -77,15 +77,14 @@ lasnormalize = function(.las, dtm)
   {
     normalized = LAS(data.table::copy(.las@data), .las@header)
 
-    lasclassify(normalized, dtm, "Zn")
```

Comment:
No. What `lasclassify` does it to call `raster::extract`, but it does it better because it updates the las object by reference